src/main/java/Eln.java

```diff
@@ -0,0 +1,47 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    static String LINE = "____________________________________________________________";
+    private static void echoUser() {
+        Scanner scan = new Scanner(System.in);
+        String input = "";
+        String[] list = new String[20];
+        int index = 0;
+
+        while(!input.equals("bye")) {
```

Comment:
Do remember to include a space between while and the condition!
I noticed this happening for your if and for loop statements down below also
```suggestion
        while (!input.equals("bye")) {
```